plugins/Satellites/src/Satellites.hpp

```diff
@@ -572,6 +615,101 @@ public slots:
 	//! Save the current satellite catalog to disk.
 	void saveCatalog(QString path=QString());
 
+	//! Set whether custom filter 'known standard magnitude' enabled.
+	//! Emits customFilterChanged() if the value changes.
+	void setFlagCFKnownStdMagnitude(bool b);
+	bool getFlagCFKnownStdMagnitude() { return Satellite::flagCFKnownStdMagnitude; }
+
+	//! Set whether custom filter 'apogee' enabled.
+	//! Emits customFilterChanged() if the value changes.
```

Comment:
Actually it emits the signal in any case. A super-clean setter should test the flag and only set+emit if it changes. So, either this (preferred), or the documentation must drop "if the value changes" (multiple times).